prac2-pages/Simranpreet Kaur/login_page.dart

```diff
@@ -0,0 +1,181 @@
+// ignore_for_file: prefer_const_literals_to_create_immutables, prefer_const_constructors
+
+import 'package:flutter/material.dart';
+import 'login2.dart' ;
+
+class LoginPage extends StatefulWidget{
+  const LoginPage({Key? key}) : super(key:key);
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+
+}
+class _LoginPageState extends State<LoginPage> {
+  bool check1 = true;
```

Comment:
It is a good practice to keep your instance variables private. (Following OOP principles)

Make sure to keep it in mind and do the necessary changes :thought_balloon: